syntax = "proto3";

package statistics;

option java_package = "com.predman.statistics";
option java_multiple_files = true;

service StatisticsService {
  rpc Predict (PredictionRequest)
  returns (PredictionReply);
}

message PredictionRequest {
  string projectId = 1;
  int32 estimatedDays = 2;
}

message PredictionReply {
  int32 predictedDays = 1;
  double estimatedDaysCertainty = 2;
}



